docs/UserGuide.md

```diff
@@ -14,27 +14,27 @@ App-Ointment is a desktop app for for managing and scheduling patient appointmen
 
 1. Ensure you have Java `11` or above installed in your Computer.
 
-1. Download the latest `App-Ointment.jar` from [here](https://github.com/AY2021S2-CS2103-W17-2/tp/releases).
+2. Download the latest `App-Ointment.jar` from [here](https://github.com/AY2021S2-CS2103-W17-2/tp/releases).
 
-1. Copy the file to the folder you want to use as the _home folder_ for your App-Ointment App.
+3. Copy the file to the folder you want to use as the _home folder_ for your App-Ointment App.
 
-1. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+4. Double-click the file to start the app. The GUI, similar to the image shown below, should appear in a few seconds. Note how the app contains some sample data.<br>
    ![Ui](images/Ui.png)
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * **`list=appt`** : Lists all appointments in the appointment schedule.
+   * **`list-appt`** : Lists all the appointments in the appointment schedule.
 
-   * **`add-patient`** `n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a patient.
+   * <code><b>add-patient</b> n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01</code> : Adds a patient to the patient records.
```

Comment:
i assume this combines the code blocks into 1 while still bolding add-patient 

(ie. avoiding this gap thing)
![image](https://user-images.githubusercontent.com/59121471/113701945-6b908580-970b-11eb-8f00-80289039920f.png)

Reply:
Yes, it'll look like this:
![image](https://user-images.githubusercontent.com/54446335/113863680-88e15480-97dc-11eb-8de5-00ac404d0261.PNG)